topics/acl.md

```diff
@@ -200,9 +207,16 @@ computers to read, while `ACL LIST` is more biased towards humans.
     5) "commands"
     6) "-@all +get"
     7) "keys"
-    8) 1) "cached:*"
+    8) "~cached:*"
     9) "channels"
-    10) 1) "*"
+    10) "&*"
+    11) "selectors"
+    12) 1) 1) "commands"
+           2) "-@all +set"
```

Comment:
commenting here, i think the doc page for ACL GETUSER must mention this change.
maybe also in the history (which is part of the json file in the redis repo)

Reply:
@madolson please make a PR to the redis-repo to add a history entry for ACL GETUSER

Reply:
Yes, that is why this one is still open :)

Reply:
https://github.com/redis/redis/pull/10200